src/core/core.tooltip.js

```diff
@@ -604,11 +608,18 @@ module.exports = function(Chart) {
 			}
 
 			ctx.fillStyle = mergeOpacity(vm.backgroundColor, opacity);
+			ctx.lineWidth = vm.borderWidth;
+			ctx.strokeStyle = vm.borderColor;
+
 			ctx.beginPath();
 			ctx.moveTo(x1, y1);
 			ctx.lineTo(x2, y2);
 			ctx.lineTo(x3, y3);
+			ctx.stroke();
```

Comment:
I thought about adding a verification `if (borderWidth > 0)` here and at line 730... Any inputs are welcome :)